src/routes/blockchains.js

```diff
@@ -154,4 +156,131 @@ router.delete(
   }
 );
 
+/* This is a post request to the blockchain route. It is using the createBlockchainValidator to
+validate the request body. It is also using the isRequired middleware to check if the user is logged
+in. If the user is not logged in, it will return a 401 error. If the user is logged in, it will
+check if the blockchain already exists. If it does not exist, it will create the blockchain. If it
+does exist, it will return a 404 error. */
+router.post('/', createBlockchainValidator, isRequired, async (req, res) => {
```

Comment:
@juniormp What is the difference between this route and the first one of the  file? Why is it necessary to do a duplication here of the route to create a new blockchain?